src/types/account.ts

```diff
@@ -133,6 +133,9 @@ export type Account = {
   // the iterated number to derive the account in a given derivationMode config
   // in context of bip44, it would be the account field of bip44 ( m/purpose'/cointype'/account' )
   index: number;
+  // a "virtual" address is a non-verified address
+  // setting this triggers the receive flow to show a warning that it can't verify the address
+  virtualAddress?: string;
```

Comment:
adding new fields at top level is costly. If there is really a need to have this, we can discuss, but generally speaking we would just introduce a namespaced `hederaResources` object (like for tronResources, bitcoinResources, etc..)